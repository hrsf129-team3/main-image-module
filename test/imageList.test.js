import React from 'react';
import ImageList from '../client/src/components/imageList.jsx';

describe('ImageList', () => {
  it('should render image list component', () => {
    const component = shallow(<ImageList allImages={[]} />);
    expect(component).toMatchSnapshot();
  });
})


// describe('List tests', () => {

//   it('renders list-items', () => {
//     const items = ['one', 'two', 'three'];
//     const wrapper = shallow(<ImageList allImages={[]} />);

//     // Expect the wrapper object to be defined
//     expect(wrapper.find('.list-items')).toBeDefined();
//     expect(wrapper.find('.item')).toHaveLength(items.length);
//   });

//   it('renders correct text in item', () => {
//     const items = ['John', 'James', 'Luke'];
//     const wrapper = shallow(<List items={items} />);

//     //Expect the child of the first item to be an array
//     expect(wrapper.find('.item').get(0).props.children).toEqual('John');
//   });
// });