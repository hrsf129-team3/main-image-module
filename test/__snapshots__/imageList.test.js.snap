// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageList should render image list component 1`] = `ShallowWrapper {}`;
